frontends/web/src/components/rates/formatCurrency.tsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2018 Shift Devices AG
+ * Copyright 2021 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { FunctionComponent } from 'react';
+import { useLoad } from '../../hooks/api';
+import { apiGet } from '../../utils/request';
+
+interface Props {
+  amount: number;
```

Comment:
this is a floating point. didn't you want to use `string`? otherwise the backend formatting doesn't do much to improve precision, does it?

Reply:
You are right, I'll change it, thx!